javascript/ql/src/semmle/javascript/frameworks/SQL.qll

```diff
@@ -98,8 +104,11 @@ private module MySql {
  * Provides classes modelling the `pg` package.
  */
 private module Postgres {
+  /** Gets an expression of the form `require('pg').Client`. */
+  API::Node newClientCallee() { result = API::moduleImport("pg").getMember("Client") }
+
   /** Gets an expression of the form `new require('pg').Client()`. */
-  API::Node newClient() { result = API::moduleImport("pg").getMember("Client").getInstance() }
+  API::Node newClient() { result = newClientCallee().getInstance() }
```

Comment:
Only used once; I suggest inlining and renaming `newClientCallee` to `newClient`.